include/libkernelflinger/log.h

```diff
@@ -38,9 +38,13 @@
 #include <vars.h>
 
 EFI_STATUS log_flush_to_var(BOOLEAN nonvol);
+EFI_FILE *efi_log_file;
+EFI_FILE *g_root;
 
 void log(const CHAR16 *fmt, ...);
 void vlog(const CHAR16 *fmt, va_list args);
+EFI_FILE *log_open_output_file(const CHAR16 *filename);
```

Comment:
Why do you need to return the file ?  You should just return an `EFI_STATUS` indicating if the operation was a success or not.

Reply:
Already modified.